test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -6,6 +6,20 @@ Describe "Tee-Object" -Tags "CI" {
 
 	$testfile = Join-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath assets) -ChildPath testfile.txt
 
+        It "Should return the output to the screen and to the variable" {
+            $teefile = $testfile
+            echo teeobjecttest1 | Tee-Object -variable teeresults
+            $teeresults         | Should -BeExactly "teeobjecttest1"
+            Remove-Item $teefile -ErrorAction SilentlyContinue
+        }
+
+        It "Should tee the output to a file" {
```

Comment:
Seems this two test don't related to the PR.

Reply:
@iSazonov yes and those are existing .

Reply:
You should keep the commit history and do rebase more accurately.

Reply:
(y) Sure and I'm learning :-)

Reply:
Yes, as of now all changes are committed.